liberty-maven-app-parent/pom.xml

```diff
@@ -45,6 +45,7 @@
                 <plugin>
                     <groupId>net.wasdev.wlp.maven.plugins</groupId>
                     <artifactId>liberty-maven-plugin</artifactId>
+                    <version>2.1.2-SNAPSHOT</version>
```

Comment:
I thought we were, but after looking at the documentation and seeing what we are doing in Jenkins, it might not be the case. http://maven.apache.org/components/maven-release/maven-release-plugin/ We are running a clean, prepare, and perform for a release, but none of these actually update the version..

Reply:
Looking at the previous change history, we think there is a good chance this will get handled correctly. JJ is aware to make sure when we do the release.